eng/Version.Details.xml

```diff
@@ -304,9 +304,9 @@
       <Sha>87d5246ddfc1fb9b07fcdf7b4b42830f67427ab9</Sha>
     </Dependency>
     <!-- Intermediate is necessary for source build. -->
-    <Dependency Name="Microsoft.SourceBuild.Intermediate.aspire" Version="9.0.0-preview.1.24079.2">
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.aspire" Version="8.0.0-preview.4.24111.3">
```

Comment:
@joperezr I assume the downgrade isn't intentional?

Reply:
this turned out to be a wrong subscription of aspire `.NET 8` channel into installer main, fixed that to `.NET 9`